org.testeditor.web.dropwizard.xtext/src/main/java/org/testeditor/web/dropwizard/xtext/validation/ValidationSummary.xtend

```diff
@@ -0,0 +1,31 @@
+package org.testeditor.web.dropwizard.xtext.validation
+
+import org.eclipse.xtend.lib.annotations.Accessors
+import org.eclipse.xtend.lib.annotations.EqualsHashCode
+import org.eclipse.xtend.lib.annotations.ToString
+
+@Accessors
+@EqualsHashCode
+@ToString
+class ValidationSummary {
+
+	var String path
```

Comment:
val ?

Reply:
would be nice, but I believe it would break serialization by Jackson (I'm assuming it relies on the default constructor to instantiate).

… I double-checked, adding simple tests for (de-)serialization. As I suspected, Jackson relies on bean-conformance :confused: